lib/dpl/provider/code_deploy.rb

```diff
@@ -39,13 +43,24 @@ def revision
         end
       end
 
+      def revision_version_info
+          s3obj = s3api.get_object({
+              bucket: option(:bucket),
+              key: s3_key,
+              range: "bytes=0-1"
```

Comment:
Is there a reason for this value? Why is it hard coded?

Reply:
B/c I don't want the object, only the metadata so it is set to the minimum.  Now I suspect I should use `head` instead of `get object`